.github/jenkinsfile/catapultDependency.groovy

```diff
@@ -0,0 +1,7 @@
+gitPullRequestPipeline {
+	dockerImageName = 'symbolplatform/build-ci:cpp-ubuntu-lts'
+	branchName = 'fix/update_catapult_dependency'
```

Comment:
change to dev on merge ?

Reply:
The PR is created to merge back into the origin branch.